crates/sqlexec/Cargo.toml

```diff
@@ -32,6 +32,10 @@ tokio-postgres = "0.7.8"
 once_cell = "1.18.0"
 url.workspace = true
 parking_lot = "0.12.1"
+serde = { workspace = true }
+
+[dependencies.sqlparser]
+features = ["serde"]
```

Comment:
What's this for?

Reply:
was looking into something & forgot to remove this.